index.html

```diff
@@ -0,0 +1,145 @@
+                <li>Features</li>
+                <li>How It Works </li>
+                <li>Pricing</li>
+            </ul>
+        </nav>
+    </header>
+
+
+    <div class="form">
+        <h2>Handcrafted, home-made masterpieces</h2>
+
+        <form action="">
+            <input placeholder="Enter your email address">
+            <button type="submit">GET STARTED</button>
+        </form>
+
+    </div>
+
+
+    <div class="desc-parent">
+
+
+        <div class="desc">
+            <i class="fa fa-3x fa-fire"></i>
+            <div class="">
+                <h2>Premium Materials</h2>
+                <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of
+                    your
+                    purchase.
+                </p>
+            </div>
+        </div>
+
+        <div class="desc">
+            <i class="fa fa-3x fa-truck"></i>
+            <div class="">
+                <h2>Fast Shipping</h2>
+                <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free
+                    returns if you are not satisfied.
+                </p>
+            </div>
+        </div>
+
+
+
+        <div class="desc">
+            <div class="">
+                <i class="fa fa-3x fa-battery-full"></i>
+            </div>
+
+
+            <div class="">
+                <h2>Quality Assurance
+                </h2>
+                <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test
+                    the pitch of your instrument.</p>
+            </div>
+        </div>
+
+
+
+
+        <div>
+            <iframe id="video" height="350px"
+                src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"
+                frameborder="0" allowfullscreen=""></iframe>
+        </div>
+    </div>
+
+
+    <section class="pricing">
+
+        <div class="card">
+            <h1>TENOR TROMBONE</h1>
+            <h2>$600</h2>
+            <P>Lorem ipsum.
+                Lorem ipsum.
+                Lorem ipsum dolor.
+                Lorem ipsum.</P>
+            <button>SELECT</button>
+        </div>
+
+
+        <div class="card">
+            <h1>BASS TROMBONE</h1>
+            <h2>$900</h2>
+            <P>Lorem ipsum.
+                Lorem ipsum.
+                Lorem ipsum dolor.
+                Lorem ipsum </P>
+            <button>SELECT</button>
+        </div>
+
+
+        <div class="card">
+            <h1>VALVE TROMBONE</h1>
+            <h2>$1200</h2>
+            <P>Plays similar to a Trumpet
+                Great for Jazz Bands
+                Lorem ipsum dolor.
+                Lorem ipsum.</P>
+            <button>SELECT</button>
+        </div>
+
+    </section>
+
+
+    <footer class="footer">
+        <ul>
+            <li>Privacy</li>
+            <li>Terms</li>
+            <li>Contact</li>
+        </ul>
+
+
+      
```

Comment:
Good job once again 👍🏽 but please refer to the other comment to see how to phrase the comments more politely. Keep going ✨ 💪🏽

Reply:
- [Optional] Good job adding the suggested changes in your other comment. But here, you have not made changes so far, Please remember it's mandatory to phrase your comment politely, Instead of `Avoid unnecessary spaces` you could say `Please remove empty spaces from your code, such practice will make your code unclear`. This is to make sure we stay humble while delivering a suggestion.